vaadin-form-layout-flow/src/test/java/com/vaadin/flow/component/formlayout/tests/FormLayoutTest.java

```diff
@@ -27,4 +27,35 @@ public void getResponsiveSteps_noInitialSteps_emptyListIsReturned() {
         FormLayout layout = new FormLayout();
         Assert.assertTrue(layout.getResponsiveSteps().isEmpty());
     }
+    
+    @Test
```

Comment:
Missing a test that actually validates that the colspan is passed to the component's element correctly, e.g. checks `componentWithColspan.getElement().getAttribute("colspan")`

Reply:
Resolved in latest commit